crypto/rand/rand_lib.c

```diff
@@ -228,11 +233,13 @@ static uint64_t get_timer_bits(void)
     }
 #else
 
-# if defined(_POSIX_C_SOURCE) \
-     && defined(_POSIX_TIMERS) \
-     && _POSIX_C_SOURCE >= 199309L \
-     && (!defined(__GLIBC__) \
-         || (defined(__GLIBC_PREREQ) && __GLIBC_PREREQ(2, 17)))
+/*
+ * Versions of glibc before 2.17 required -lrt to be included to access the
+ * POSIX timer functions.  To avoid adding a dependency, we test for the
+ * version as well as the presence of the timers.
+ */
```

Comment:
I have general English language question. And do note that this is a *question*. I'd argue that _POSIX_TIMERS check is "primordial" in below condition. Does "as well" formulation above preserves this impression? I mean to me above sounds more like glibc version check is the prime purpose of exercise, which kind of stings the eye.

[BTW, another thing that stings the eye is "version of glibc". I mean I'd rather say "glibc version". I suppose this is likely to be American vs. non-American thing...]

Reply:
The _versions of glibc_ vs _glibc versions_ could go either way.

I used the apostrophe free genitive.  The more normal genitive: _glibc's versions_ wouldn't be correct.  You are correct though that _glibc versions_ would be fine.